components/collapsible-panel/test/collapsible-panel-group.test.js

```diff
@@ -0,0 +1,41 @@
+import '../collapsible-panel.js';
+import '../collapsible-panel-group.js';
+import { expect, fixture, html, runConstructor } from '@brightspace-ui/testing';
+
+describe('d2l-collapsible-panel-group', () => {
+
+	it('should construct', () => {
+		runConstructor('d2l-collapsible-panel-group');
+	});
+
```

Comment:
These could have been vdiffs but I feel like is a good place to try to avoid adding more!

Reply:
Yep, we already test that both `default` and `inline` look as expected, right? So, no need to check these visually.

Reply:
We do yeah.